test/integration/passenger/controls/repository.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+case platform.family
+when 'redhat'
+  repo_file = '/etc/yum.repos.d/passenger.repo'
+  repo_url = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/$releasever/$basearch'
+when 'debian'
+  # Inspec does not provide a `codename` matcher, so we add ours
+  finger_codename = {
+    'ubuntu-18' => 'bionic',
+    'ubuntu-20' => 'focal',
```

Comment:
Apologies, this is supposed to be:

```suggestion
    'ubuntu-18.04' => 'bionic',
    'ubuntu-20.04' => 'focal',
```

* https://gitlab.com/myii/nginx-formula/-/pipelines/462846132

I tried pushing the changes here but perhaps edits by maintainers has been disabled?